ex2_hal/charon/hardware_interface/source/skytraq_gps.c

```diff
@@ -84,7 +126,11 @@ GPS_RETURNSTATE gps_configure_message_types(uint8_t GGA, uint8_t GSA, uint8_t GS
     GSA_ENABLED = GSA ? true : false;
     GSV_ENABLED = GSV ? true : false;
     RMC_ENABLED = RMC ? true : false;
+#if FLIGHT_CONFIGURATION == 1
```

Comment:
This symbol should distinguish between GPS types, not an esoteric "flight configuration"